src/olympia/conf/dev/settings.py

```diff
@@ -47,13 +47,13 @@
 
 DATABASES = {
     'default': get_db_config('DATABASES_DEFAULT_URL', charset='utf8mb4'),
-    'slave': get_db_config(
+    'replica': get_db_config(
         'DATABASES_SLAVE_URL', atomic_requests=False, charset='utf8mb4'),
```

Comment:
I guess we should change the DATABASES_SLAVE_URL` to `DATABASES_REPLICA_URL` to keep consistency @eviljeff Let me know if we need to it as well. Thanks.

Reply:
Sure.  It's another external env variable so we'll need to ops to update those on their end too.